ApollosProject/apollos-prototype
# Removes Duplicate Copyright Info

## DESCRIPTION

![Screen Shot 2019-11-18 at 11 29 34 AM](https://user-images.githubusercontent.com/2659478/69071082-4f495700-09f7-11ea-830a-b281682d6913.png)


### What does this PR do, or why is it needed?

This removes duplicate copyright info from showing up on devotionals and exports `LegalText` for the client to use.

### What design trade-offs/decisions were made?

Will only show the copyright from the *first* scripture in the array. Not a problem right now but if we ever show scriptures from multiple translations in a devo, we'll have to figure it out.

### How do I test this PR?

Temporailily turn off the "PUBLIC DOMAIN" check (that's what I did for the screenshot)

### What automated tests did you write?

None.

## TODO:

- [x] Closes [#1213]

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
